src/main/resources/static/user.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
file should be removed and changes should be merged to /template/user.html instead of /static/user.html